advisor/src/main/kotlin/advisors/Osv.kt

```diff
@@ -200,8 +200,7 @@ private fun Vulnerability.toOrtVulnerability(): org.ossreviewtoolkit.model.Vulne
     }
 
     val references = references.mapNotNull { reference ->
-        var url = reference.url.trim()
-        url = url.takeUnless { it.startsWith("://") } ?: "https$url"
+        val url = reference.url.trim().let { url -> url.takeUnless { it.startsWith("://") } ?: "https$url" }
```

Comment:
Unrelated, but while at it you could also replace `takeUnless` with a normal `if`, because it's shorter and easier to understand:
```
if (url.startsWith("://")) "https$url" else url
```